### PR TITLE
[NFT_Game] Change "you your" to "show your"

### PR DESCRIPTION
**Pull request by quiint**
"...OpenSea or Rarible should you your NFT w/..." doesn't make sense, so I changed it to "...OpenSea or Rarible should show your NFT w/..." 🙂

### CODE REVIEW

File: NFT_Game/en/Section_2/Lesson_2_Deploy_And_See_NFTs_Changing.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ I deploy using `npx hardhat run scripts/deploy.js --network rinkeby`. From there
 Contract deployed to: 0x02f59Dc14666c4480Ae4b477eFfF15949970dfeA 
 ```
 
-Once you wait a few min, sites like OpenSea or Rarible should you your NFT w/ it's update HP #.
+Once you wait a few min, sites like OpenSea or Rarible should show your NFT w/ it's update HP #.
```
Comment (adilanchian) [inline NFT_Game/en/Section_2/Lesson_2_Deploy_And_See_NFTs_Changing.md:65]: can you add an 's' at the end of min here?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Good catch :)

Thanks!!

File: NFT_Game/en/Section_2/Lesson_2_Deploy_And_See_NFTs_Changing.md (hunk 1 above)
Comment (quiint) [reply NFT_Game/en/Section_2/Lesson_2_Deploy_And_See_NFTs_Changing.md]: All fixed! 🙂

**Comment by adilanchian** [comment]
Hey @quiint !

Thanks for this :). All good to go!

**Comment by adilanchian** [comment]
@all-contributors please add @quiint  for doc

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION

✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨

**Comment by quiint** [comment]
> ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION
> 
> 
> 
> ✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨

Super excited 🎉

**Pull request merged**